apps/aecore/src/aec_sync_stats.erl

```diff
@@ -161,7 +161,7 @@ set_sync_stat(Id, SyncStat, State) ->
     lists:keystore(Id, #sync_stat.id, State, touch(SyncStat)).
 
 touch(SyncStat) ->
-    SyncStat#sync_stat{ last_update = now_ms() }.
+    SyncStat#sync_stat{ last_update = erlang:system_time(millisecond) }.
```

Comment:
Why not monotonic here?

Reply:
I just inlined a single use utility function here. Changing the api call could change the meaning of  the 'touch' function, and I don't want to do that unless I know exactly how it's used.

Reply:
I am pretty sure it is only used for (internal) statistics so monotonic or not makes little difference.